xrtl/port/linux/ui/BUILD

```diff
@@ -44,6 +44,7 @@ cc_test(
         "//xrtl/tools/target_config",
     ],
     size = "small",
+    tags = ["requires_gui"],
```

Comment:
will throw them in README for now until we have enough stuff to warrant other docs